falguard.py

```diff
@@ -64,7 +65,11 @@ class Validator:
 
     def __init__(self, spec_path):
         with open(spec_path, 'r') as f:
-            spec_dict = json.loads(f.read())
+            ext = spec_path.split(".")[-1]
```

Comment:
Let's use recommended `os.path.splitext` here
https://stackoverflow.com/a/541394

Reply:
Used to but the `filename` will not be used so the linter will tells it's wrong. Or is there a workaround to avoid this?